docs/src/usage/usage.rst

```diff
@@ -43,60 +33,38 @@ Running the interactive shell
                    pyspark --py-files ${OPAQUE_HOME}/target/scala-2.12/opaque_2.12-0.1.jar  \
                      --jars ${OPAQUE_HOME}/target/scala-2.12/opaque_2.12-0.1.jar
     
-   (we need to specify `--py-files` is because the Opaque Python functions are placed in the .jar for easier packaging)
+   (we need to specify `--py-files` because the Python functions are placed in the .jar for easier packaging)
+
+3. Alternatively, you can also run queries in Scala locally using ``sbt``.
+
+   .. code-block:: bash
+
+                   build/sbt console
     
-3. Inside the Spark shell, import Opaque's DataFrame methods and install Opaque's query planner rules.
+4. Inside the Spark shell, import Opaque SQL's ``DataFrame`` methods and its query planning rules.
 
    Scala:
 
    .. code-block:: scala
 
                      import edu.berkeley.cs.rise.opaque.implicits._
-                     edu.berkeley.cs.rise.opaque.Utils.initSQLContext(spark.sqlContext)
+                     edu.berkeley.cs.rise.opaque.Utils.initOpaqueSQL(spark)
 
    Python:
 
    .. code-block:: python
 
                   from opaque_sql import *
-                  init_sql_context()
+                  init_opaque_sql()
                    
     
 
-4. Alternatively, you can also run queries in Scala locally.
-
-   .. code-block:: bash
-
-                   cd ${OPAQUE_HOME}
-                   JVM_OPTS="-Xmx4G"; build/sbt console
-
-
-Starting the gRPC Listener
-**************************
-
-1. To run locally for easy testing/functionality:
-
-   .. code-block:: bash
-
-                   build/sbt run
-
-2. To launch the listener on a standalone Spark cluster:
-
-   .. code-block:: bash
-
-                  build/sbt assembly # create a fat jar with all necessary dependencies
-
-                  spark-submit --class edu.berkeley.cs.rise.opaque.rpc.Listener  \
-                     <Spark configuration parameters> \
-                     --deploy-mode client ${OPAQUE_HOME}/target/scala-2.12/opaque-assembly-0.1.jar
```

Comment:
Is this no longer needed? I thought this needs to stay because we also want to interface with the mc2 client?

Reply:
Forgot to add the new page for this to Git